src/cms/enhanced-media-gallery-license-image.md

```diff
@@ -0,0 +1,36 @@
+---
+title: License an Adobe Stock Image
+---
+
+[Enhanced Media Gallery]({% link cms/enhanced-media-gallery.md %}) provides a direct integration with Adobe Stock, allowing the user to license their images directly from the gallery page.
```

Comment:
Please mention that this functionality requires Adobe Stock Integration to be enabled and configured